salt/serializers/yamlex.py

```diff
@@ -129,17 +122,14 @@
 
 # prefer C bindings over python when available
 BaseLoader = getattr(yaml, "CSafeLoader", yaml.SafeLoader)
-if six.PY3:
-    # CSafeDumper causes repr errors in python3, so use the pure Python one
-    try:
-        # Depending on how PyYAML was built, yaml.SafeDumper may actually be
-        # yaml.cyaml.CSafeDumper (i.e. the C dumper instead of pure Python).
-        BaseDumper = yaml.dumper.SafeDumper
-    except AttributeError:
-        # Here just in case, but yaml.dumper.SafeDumper should always exist
-        BaseDumper = yaml.SafeDumper
-else:
-    BaseDumper = getattr(yaml, "CSafeDumper", yaml.SafeDumper)
+# CSafeDumper causes repr errors in python3, so use the pure Python one
+try:
+    # Depending on how PyYAML was built, yaml.SafeDumper may actually be
+    # yaml.cyaml.CSafeDumper (i.e. the C dumper instead of pure Python).
+    BaseDumper = yaml.dumper.SafeDumper
+except AttributeError:
+    # Here just in case, but yaml.dumper.SafeDumper should always exist
+    BaseDumper = yaml.SafeDumper
```

Comment:
**question** did the comments get out of date with the api? i.e. `yaml.cyaml.CSafeDumper` in the comment vs `yaml.dumper.SafeDumper` in the code?